README.md

```diff
@@ -8,7 +8,8 @@
 
 ```swift
 struct MainPage: HTMLDocument {
-    var title: String = "Elementary"
+    var title = "Elementary"
+    var lang = "en"
```

Comment:
I don't think it should be in the introduction snipped, keep it as simple as possible